appformer-client-api/src/main/java/org/appformer/client/keyboardShortcuts/KeyboardShortcutsApiOpts.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.appformer.client.keyboardShortcuts;
+
+import jsinterop.annotations.JsProperty;
+import jsinterop.annotations.JsType;
+
+@JsType
+public class KeyboardShortcutsApiOpts {
+
+    public static final KeyboardShortcutsApiOpts DEFAULT = new KeyboardShortcutsApiOpts(Repeat.NO_REPEAT);
+
+    private final Repeat repeat;
+
+    public KeyboardShortcutsApiOpts(final Repeat repeat) {
+        this.repeat = repeat;
+    }
+
+    @JsProperty
+    public boolean getRepeat() {
+        return Repeat.REPEAT.equals(repeat);
+    }
+
+    public enum Repeat {
+        REPEAT,
+        NO_REPEAT
+    }
```

Comment:
Could we please add javadoc to `KeyboardShortcutsApiOpts`. To me as newcomer is not clear what `Opts` stands for and what is the difference between `REPEAT` and `NO_REPEAT` constants. Thank you.

Reply:
Added JavaDoc.

Reply:
Thank you